language/German/strings.po

```diff
@@ -805,7 +811,7 @@ msgstr "Sound"
 #: src/frontend/mame/ui/info.cpp:46
 msgctxt "emulation-feature"
 msgid "capture hardware"
-msgstr ""
+msgstr "Aufzeichnungsgerät"
```

Comment:
"Aufzeichnungshardware"? it's not "capture device". But I guess it could be synonymous in this case....

Reply:
I'd rather stick with _gerät_ here